### ISSUE TITLE
jszip-cli: no files were compressed

### ISSUE THREAD

**Issue by Rebuilding127** [ffflorian/node-packages#697]
Here is my script: `jszip-cli add -o ./dist.zip ./dist/`

When I use the package `"@ffflorian/jszip-cli": "^3.6.2"`, it works.
But when I use the latest version `"@ffflorian/jszip-cli": "^3.7.1"`, it not works.

Why?

**Comment by ffflorian** [comment]
@Rebuilding127 thanks for the report, I'll check.

**Comment by canstand** [comment]
Same issue. `-V` show this: 

```
ℹ️  jszip-cli/BuildService TypeError: fs.lstat is not a function
    at BuildService.checkEntry (file:///C:/Users/xxx/node_modules/.pnpm/@ffflorian+jszip-cli@3.7.1_typescript@5.5.4/node_modules/@ffflorian/jszip-cli/dist/BuildService.js:118:33)
```

**Comment by ffflorian** [comment]
Fixed in 47057ec796d67e2ba9710f4bc5423d4f840eee0c and released in v3.7.2.
